templates/releases_feed.hbs

```diff
@@ -7,7 +7,7 @@
 <link href="https://pubsubhubbub.appspot.com" rel="hub" />
 <link href="https://pubsubhubbub.superfeedr.com" rel="hub" />
 <id>urn:docs-rs:{{cratesfyi_version_safe}}</id>
-<updated>{{content[0].release_time_rfc3339}}</updated>
+<updated>{{content.[0].release_time_rfc3339}}</updated>
```

Comment:
```suggestion
<updated>{{content.0.release_time_rfc3339}}</updated>
```
I'm not sure that `.[0]` is entirely correct syntax

Reply:
It works correctly, why would `.0` be better?

Reply:
If it works then it's up to you what you want to do, I've got no idea if one is objectively better or not